.github/workflows/get-prettyurl-conf.yml

```diff
@@ -0,0 +1,29 @@
+name: get prettyurl conf
+
+on:
+  push:
+    branches:    
+    - '**'
+    - '!master'
+    - '![12][0-9]'
+
+jobs:
+  get-prettyurl-conf:
+    name: Get Prettyurl conf
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout the repository
+        uses: actions/checkout@v2
+      - name: Create local changes
+        run: |
+          ./prettyurl.sh
+      - name: Commit files
+        run: |
+          git config --local user.email "41898282+github-actions[bot]@users.noreply.github.com"
+          git config --local user.name "github-actions[bot]"
+          git commit -m "Add prettyurl conf" -a
+      - name: Push changes
+        uses: ad-m/github-push-action@master
+        with:
+          github_token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
to make this new workflow work, someone with configure rights on the repository needs to add a GITHUB_TOKEN secret. See https://docs.github.com/en/actions/reference/encrypted-secrets#creating-encrypted-secrets-for-a-repository. It needs to be called GITHUB_TOKEN. Otherwise I could change its name.
@kyrofa Could you do this?

Reply:
it seems like this has worked also without adding the token